deploy/scripts/000101-transfer-proxy-admin-ownership.ts

```diff
@@ -0,0 +1,21 @@
+import { execute, InstanceName, isLive, setDeploymentMetadata } from '../../utils/Deploy';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
+    const { deployer, daoMultisig } = await getNamedAccounts();
+
+    await execute({
+        name: InstanceName.ProxyAdmin,
+        methodName: 'transferOwnership',
```

Comment:
We should consider an `Ownable2Step`-based (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable2Step.sol) `ProxyAdmin`

Reply:
I agree, even though this isn't in the PR's scope

Reply:
👍 

@yudilevi FYI